src/apps/common/submodules/portWizard/portWizard.js

```diff
@@ -478,13 +484,109 @@ define(function(require) {
 		 * @param  {Function} callback  Callback to pass the step template to be rendered
 		 */
 		portWizardCarrierSelectionRender: function(args, callback) {
-			var self = this;
+			var self = this,
+				formattedNumbers = _.get(args.data, 'nameAndNumbers.numbersToPort.formattedNumbers'),
+				numbers = _.map(formattedNumbers, 'e164Number'),
+				initTemplateMultiple = function(dataTemplate) {
+					dataTemplate.errorType = 'noWinnerCarriers';
```

Comment:
```suggestion
					dataTemplate.errorType = 'noWinningCarriers';
```

Also, you should use `merge#lodash` to create a new object rather than mutating the existing one.